crates/swift-bridge-ir/src/codegen/generate_c_header.rs

```diff
@@ -148,6 +148,12 @@ typedef struct {option_ffi_name} {{ bool is_some; {ffi_name} val; }} {option_ffi
                             variants += &variant;
                         }
 
+                        let derive_debug_impl = if ty_enum.derive.debug {
+                            format!("void* {ffi_name}$_Debug({ffi_name} this);")
```

Comment:
Should end the string with a newline `\n`.

Reply:
It generates just fine:
```h
// File automatically generated by swift-bridge.
#include <stdbool.h>
typedef enum __swift_bridge__$DeriveDebugEnumTag { __swift_bridge__$DeriveDebugEnum$Variant, } __swift_bridge__$DeriveDebugEnumTag;
typedef struct __swift_bridge__$DeriveDebugEnum { __swift_bridge__$DeriveDebugEnumTag tag; } __swift_bridge__$DeriveDebugEnum;
typedef struct __swift_bridge__$Option$DeriveDebugEnum { bool is_some; __swift_bridge__$DeriveDebugEnum val; } __swift_bridge__$Option$DeriveDebugEnum;
void* __swift_bridge__$DeriveDebugEnum$_Debug(__swift_bridge__$DeriveDebugEnum this);
void* __swift_bridge__$Vec_DeriveDebugEnum$new(void);
void __swift_bridge__$Vec_DeriveDebugEnum$drop(void* vec_ptr);
void __swift_bridge__$Vec_DeriveDebugEnum$push(void* vec_ptr, __swift_bridge__$DeriveDebugEnum item);
__swift_bridge__$Option$DeriveDebugEnum __swift_bridge__$Vec_DeriveDebugEnum$pop(void* vec_ptr);
__swift_bridge__$Option$DeriveDebugEnum __swift_bridge__$Vec_DeriveDebugEnum$get(void* vec_ptr, uintptr_t index);
__swift_bridge__$Option$DeriveDebugEnum __swift_bridge__$Vec_DeriveDebugEnum$get_mut(void* vec_ptr, uintptr_t index);
uintptr_t __swift_bridge__$Vec_DeriveDebugEnum$len(void* vec_ptr);
void* __swift_bridge__$Vec_DeriveDebugEnum$as_ptr(void* vec_ptr);
```